portal-api/api-tests/v1/gef/facilities.api-test.js

```diff
@@ -68,7 +68,8 @@ describe(baseUrl, () => {
         coverDateConfirmed: null,
         canResubmitIssuedFacilities: null,
         issueDate: null,
-        unissuedToIssuedByMaker: expect.any(Object)
+        unissuedToIssuedByMaker: expect.any(Object),
+        hasBeenIssuedAndAcknowledged: null,
```

Comment:
do we need this to be null? or can we set it to false? I assume it's always false until changed to `true`

Reply:
it can be false, but is null when application created and thought to keep it in-line with all the others